bigquery_etl/cli/__init__.py

```diff
@@ -1,22 +1,29 @@
 """bigquery-etl CLI."""
 
 import click
+import warnings
 
 from ..cli.query import query
 from ..cli.dag import dag
+from ..cli.dryrun import dryrun
+from ..cli.format import format
 from .._version import __version__
 
 
 def cli():
     """Create the bigquery-etl CLI."""
-    commands = {"query": query, "dag": dag}
+    commands = {"query": query, "dag": dag, "dryrun": dryrun, "format": format}
 
     @click.group(commands=commands)
     @click.version_option(version=__version__)
     def group():
         """CLI tools for working with bigquery-etl."""
         pass
 
+    warnings.filterwarnings(
+        "ignore", "Your application has authenticated using end user credentials"
+    )
```

Comment:
When authenticating via `gcloud auth default`, every time `google.cloud.bigquery.Client` makes a request the following warning shows up: 

```
UserWarning: Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/
  warnings.warn(_CLOUD_SDK_CREDENTIALS_WARNING)
```

This mutes this warning when using the CLI